src/pray.c

```diff
@@ -1417,6 +1417,7 @@ dosacrifice()
                 pline_The("altar is stained with %s blood.", urace.adj);
                 levl[u.ux][u.uy].altarmask = AM_CHAOTIC;
                 angry_priest();
+                tnnt_achieve(A_CONVERTED_ALTAR);
```

Comment:
I'm not sure this one should apply unless you're chaotic yourself. A lawful turning a neutral altar chaotic isn't really "converting" it.

Reply:
the achievement just says "convert an altar", not "convert it to your own alignment." but yeah, i can throw an `if (u.r.totes.chaotic)` in there!  the achievement should maybe be named more clearly, though.